### PR TITLE
Feature/11.student portfolio updated

### PR DESCRIPTION
**Pull request by mateuszslowinski**
Updated for student portfolio. Changes shows Student Portfolio component to modal view instead route view.

### CODE REVIEW

**Review by MS151994 — DISMISSED** [review]
LGFM :)

File: src/App.tsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { AdminPage } from './pages/Admin.page'
 import { NotFoundPage } from './pages/NotFound.page'
 import { LoginPage } from './pages/Login.page'
 import { HrPage } from './pages/Hr.page'
-import { StudentCVPage } from './pages/StudentCV.page'
```
Comment (Fremen1990) [inline src/App.tsx:9]: 👍

File: src/components/StudentPortfolio/AsideUserBox/AsideUserBox.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const AsideUserBox = (props: Props) => {
             src="https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_640.png"
             alt="avatar użytkownika"
          />
-         <h2>Jan Kowalski</h2>
+         <h2>jan kowalski</h2>
```
Comment (Fremen1990) [inline src/components/StudentPortfolio/AsideUserBox/AsideUserBox.tsx:31]: Drastic change :D

File: src/components/StudentPortfolio/ExpectationBox/ExpectationBox.tsx [hunk 3]
```diff
@@ -18,7 +18,6 @@ export const ExpectationBox = () => {
    return (
       <>
          <SubtitlesSection text={expectation} />
-
```
Comment (Fremen1990) [inline src/components/StudentPortfolio/ExpectationBox/ExpectationBox.tsx:21]: Please use linter and pretier to remove this emply lines. It is configured in project, but maybe its disabled in your local.

File: src/components/StudentPortfolio/RatingBox/RatingViewBox/RatingViewBox.tsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ interface Props {
 export const RatingViewBox = (props: Props) => {
    return (
       <RatingViewBoxContainer>
-         <p>{props.title}</p>
+         <h2>{props.title}</h2>
```
Comment (Fremen1990) [inline src/components/StudentPortfolio/RatingBox/RatingViewBox/RatingViewBox.tsx:17]: Destructuring for props is needed

File: src/components/commons/Button/Button.tsx [hunk 5]
```diff
@@ -3,8 +3,9 @@ import { PrimaryButton } from './Button.styles'
 
 interface Props {
    buttonTitle: string
+   method?: () => void
 }
 
-export const Button = (props: Props) => {
-   return <PrimaryButton>{props.buttonTitle}</PrimaryButton>
+export const Button = ({ method, buttonTitle }: Props) => {
+   return <PrimaryButton onClick={method}>{buttonTitle}</PrimaryButton>
```
Comment (Fremen1990) [inline src/components/commons/Button/Button.tsx:10]: Destructurization 👍

File: src/components/recruiters/OneUser/OneUser.tsx [hunk 6]
```diff
@@ -17,90 +18,109 @@ interface Props {
...
+                     <InterViewMeBox
+                        ratingBox={false}
+                        boxTitle={descriptions.practice}
+                        point={'tak'}
+                     />
+                     <InterViewMeBox
+                        ratingBox={false}
+                        boxTitle={descriptions.experience}
+                        point={'6 mies'}
+                     />
```
Comment (Fremen1990) [inline src/components/recruiters/OneUser/OneUser.tsx:119]: Pull the point from API as well as the rest of student data

File: src/constants/description/description.ts [hunk 7]
```diff
@@ -10,17 +10,17 @@ export const description = {
...
+      courseEvaluation: 'ocena przejścia kursu',
+      activityAssessment: 'ocena aktywności i zaangażowania na kursie',
+      codeEvaluation: 'ocena kodu w projekcie własnym',
+      scrumAssessment: 'ocena pracy w zespole Scrum',
+      preferWork: 'preferowane miejscie pracy',
+      targetPlace: 'docelowe miasto, gdzie kandydat chce pracować',
+      contractType: 'oczekiwany typ kontraktu',
+      targetSalary: 'oczekiwanie miesięcczne wynagrodzenie netto',
+      practice: 'zgoda na odbycie bezpłatnych praktyk/stażu na początek',
+      experience: 'komercyjne dościadcznie w programowaniu',
```
Comment (Fremen1990) [inline src/constants/description/description.ts:23]: Great idea!! 🥇

**Review by Fremen1990 — APPROVED** [review]
Some changes needed, but we will do this in the final refactorization with other same kind corrections 👍

**Pull request merged**